### PR TITLE
QAE-216 - Test case for submenu border color, radius and width option

### PR DESCRIPTION
**Pull request by bsfprajaktap**
Description
Header builder -> Primary menu -> Submenu border color, width and radius

Screenshots

How has this been tested?
npm run test:e2e:interactive tests/e2e/specs/customizer/header-builder/primary-menu-submenu/border-color-radius-width.test.js
Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/utils/create-menu.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import { createURL, createNewPost, publishPost } from '@wordpress/e2e-test-utils';
```
Comment (bsfsnehal) [inline tests/e2e/utils/create-menu.js:1]: check for "publish-post" util path

File: tests/e2e/specs/customizer/header-builder/primary-menu-submenu/primary-menu-submenu-border.test.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+			`${ submenuBorder[ 'header-menu1-submenu-border' ].bottom + 'px' }`,
+		);
+		await expect( {
+			selector: '.sub-menu',
+			property: 'border-right-width',
+		} ).cssValueToBe(
+			`${ submenuBorder[ 'header-menu1-submenu-border' ].right + 'px' }`,
+		);
+	} );
+} );
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/header-builder/primary-menu-submenu/primary-menu-submenu-border.test.js:63]: file name should be as "border.test"

Comment (bsfsnehal) [inline tests/e2e/specs/customizer/header-builder/primary-menu-submenu/primary-menu-submenu-border.test.js:63]: test case name be as "border-width.test.js"

File: tests/e2e/specs/customizer/header-builder/primary-menu-submenu/border-width.test.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
+import { createURL } from '@wordpress/e2e-test-utils';
+import { createNewMenu } from '../../../../utils/create-menu';
+import { setCustomize } from '../../../../utils/customize';
+describe( 'Add sub menu for primary menu and add border to the sub menu', () => {
+	it( 'sub menu should be added successfully', async () => {
+		await createNewMenu();
+		//center alignment for desktop, tablet and mobile
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/header-builder/primary-menu-submenu/border-width.test.js:7]: remove unwanted comment

**Review by bsfsnehal — APPROVED** [review]

File: tests/e2e/specs/customizer/header-builder/primary-menu-submenu/border-color-radius-width.test.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+			property: 'border-color',
+		} ).cssValueToBe( `${ submenuBorder[ 'header-menu1-submenu-b-color' ] }` );
+
+		await expect( {
+			selector: '.ast-builder-menu-1 .sub-menu',
+			property: 'border-radius',
+		} ).cssValueToBe( `${ submenuBorder[ 'header-menu1-submenu-border-radius' ] + 'px' }` );
+
+		await expect( {
+			selector: '.sub-menu',
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/header-builder/primary-menu-submenu/border-color-radius-width.test.js:40]: Can we verify selector for all width:- https://share.bsf.io/yAuKg7wQ

Comment (SimranChougule1) [reply tests/e2e/specs/customizer/header-builder/primary-menu-submenu/border-color-radius-width.test.js:40]: changed selector

**Review by GauravKhupse — APPROVED** [review]

**Pull request merged**